pkg/output/stdout.go

```diff
@@ -164,7 +165,10 @@ func (s *Stdoutput) Progress(status ffuf.Progress) {
 	dur -= mins * time.Minute
 	secs := dur / time.Second
 
-	fmt.Fprintf(os.Stderr, "%s:: Progress: [%d/%d] :: Job [%d/%d] :: %d req/sec :: Duration: [%d:%02d:%02d] :: Errors: %d ::", TERMINAL_CLEAR_LINE, status.ReqCount, status.ReqTotal, status.QueuePos, status.QueueTotal, reqRate, hours, mins, secs, status.ErrorCount)
+	percentage := int(math.Round((float64(status.ReqCount) / float64(status.ReqTotal)) * 100))
+	percentage_str := strconv.Itoa(percentage) + "%"
+
+	fmt.Fprintf(os.Stderr, "%s:: Progress: [%d/%d] %s :: Job [%d/%d] :: %d req/sec :: Duration: [%d:%02d:%02d] :: Errors: %d ::", TERMINAL_CLEAR_LINE, status.ReqCount, status.ReqTotal, percentage_str, status.QueuePos, status.QueueTotal, reqRate, hours, mins, secs, status.ErrorCount)
```

Comment:
Hi @pry0cc,

As this is a hot code path, I think we might want to avoid the Itoa call if possible. What do you think about this?

```suggestion
	percentage := int(math.Round((float64(status.ReqCount) / float64(status.ReqTotal)) * 100))
	
	fmt.Fprintf(os.Stderr, "%s:: Progress: [%d/%d] %d%% :: Job [%d/%d] :: %d req/sec :: Duration: [%d:%02d:%02d] :: Errors: %d ::", TERMINAL_CLEAR_LINE, status.ReqCount, status.ReqTotal, percentage, status.QueuePos, status.QueueTotal, reqRate, hours, mins, secs, status.ErrorCount)
```

What do you think?

Reply:
I have no confidence in my go abilities, and I'm glad you have another solution! Looks awesome!